Joplin_install_and_update.sh

```diff
@@ -11,6 +11,14 @@ echo "              | |                                                 "
 echo "              |_|                                                 "
 echo ""
 
+# Check and warn if running as root.
+if [[ $EUID = 0 ]] ; then
+  if [[ $* != *--allow-root* ]] ; then
+    echo "Please don't run as root. Use '--allow-root' to ignore this warning."
```

Comment:
Of course.
If the ~/.joplin directory is owned by root:root, and the script is subsequentially run by a non-root user (for example, to update), it will fail since it is not allowed to change anything in the directory.

You can of course then just run it again as root, but that may not be possible, and will encourage giving unnecessary privileges to some internet-script.

Reply:
Yes that makes sense and I agree that it's better not to install it as root since it's not needed. Maybe let's simply put "It is not recommanded (nor necessary) to run this script as root. To do it anyway, please use '--allow-root'". What do you think?

Reply:
I've made some small changes to your text and created a new commit.